### DISCUSSION TITLE
Using SelectIntegrationFeatures() on scATAC data before integrating?

### DISCUSSION THREAD

**Discussion by danielcgingerich** [stuart-lab/signac#515] [Ideas]
The scATAC integration vignette constructs a common peak set among 2 or more scATAC datasets.  Then, peaks are randomly sampled from this set and used for integration.  

I was wondering if this is a better alternative: 
Run `FindTopFeatures() ` and set `min.cutoff = 'q0'` on each dataset.  Then run `SelectIntegrationFeatures()` and set `nfeatures =` however many features you want.  As long as this number is less than the number of common peaks between each dataset, the features used will be in common between all datasets.  More so, they will be ranked instead of randomly sampled, so you will be using the most informative features to integrate.

**Comment by timoast** [comment]
Sure, you could do that. The random sampling approach is not a very good solution and we're working on some better, more scalable solutions for scATAC-seq integration that should be available soon.

**Reply by danielcgingerich** [reply]
Ok, thanks.  I wasn't sure if SelectIntegrationFeatures() could be used for ATAC data
